cli/config.go

```diff
@@ -49,6 +50,16 @@ var configCommand = cli.Command{
 	},
 }
 
+func writeOutput(outputDir string, scanResult models.ScanResultModel, format output.Format) (string, error) {
```

Comment:
writeOutput seems too mainstream, it could be for example: writeScanResultOutput